frontend/app/src/types/frontend-settings.ts

```diff
@@ -176,7 +176,7 @@ export const FrontendSettings = z.object({
       Defaults.DEFAULT_VERSION_UPDATE_CHECK_FREQUENCY
     )
   ),
-  enableEthNames: z.boolean().default(true)
+  enableAliasNames: z.boolean().default(true)
```

Comment:
The only problem with renaming the property is that the users will have to set it from scratch since we don't do any migration.

Reply:
should we keep the old state name?

Reply:
I was thinking that it's okay since the setting is easy to toggle, and also the majority will keep the value true.